doc/update063.md

```diff
@@ -0,0 +1,31 @@
+### dPoW 0.6.3 update information
+
+#### Update seednode mm2
+
+Please update your seednodes to [`0f6c72615`](https://github.com/KomodoPlatform/atomicDEX-API/releases/download/beta-2.1.7404/mm2-0f6c72615-Linux-Release.zip).
+Once updated, stop and restart your seednode(s). Make sure to check your certificates have not expired! 
+If you see an error like `13 12:54:37, mm2:394] mm2:457] mm2:276] lp_native_dex:355] lp_native_dex:589] P2P initializing error: 'Error reading WSS key/cert file "/etc/letsencrypt/live/na3p.smk.dog/privkey.pem": Permission denied (os error 13)'` in your logs, check the instructions at https://github.com/smk762/nn_mm2_seed#step-4-restart-mm2 to fix the permissions.
+
+```bash
+cd ~/nn_mm2_seed
+git pull
+./stop_mm2.sh
+./update_mm2.sh
```

Comment:
https://github.com/KomodoPlatform/atomicDEX-API/releases/tag/beta-2.1.8388
breaks update_mm2.sh because it's only Darwin and the script parses to the latest only

Reply:
Builds for other OS are incoming. If not available in time for this update I can use a CI  build link, though this wont work for some and they'll need to build from source.. Fortunately its not too hard.